test/scripts/set_solutions_independent_mode_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+class SetSolutionsIndependentModeTest < ActiveSupport::TestCase
+  test "it populates correctly" do
+    skip
```

Comment:
Should this be skipped? If so, would you add the optional message to explain why?

```
skip "this is expensive to run, unskip it if making changes to the whatsit feature"
```

Reply:
All three scripts should be commented out now. I want to keep them around in case we need them again but don't want to invest in keeping them up to date with the codebase each time we change something. Will add skips as you suggest and also a comment with a "Can be deleted after XYZ"